config/routes.rb

```diff
@@ -466,4 +466,6 @@ def redirect(path, options = { prefix: Whitehall.router_prefix })
   mount Flipflop::Engine => "/flipflop"
 
   mount GovukPublishingComponents::Engine, at: "/component-guide" unless Rails.env.production?
+
+  mount ObjectStore::Engine, at: "/object-store"
```

Comment:
I _think_ we'll need to mount the Engine inside the `government/admin` namespace here. I'm not sure why we do that (perhaps @ryanb-gds could advise?), but it seems like most of the functionality is there.

Reply:
It's a legacy thing, Whitehall used to render public facing GOV.UK pages, and the publishing app was at `government/admin`. But the frontend stuff was broken out into a separate app, `government-frontend`

Reply:
Ah ha - so we don't necessarily _need_ to route within `/government/admin` then?

Reply:
I'm not sure, to be honest. I think there are probably things like layouts, maybe some middlewares, that are automatically applied at that path. But if there's a good reason not to use it then I expect you could make it work.